### DISCUSSION TITLE
Is fields object still required as defaultValue when using useWatch since v6.11.0?

### DISCUSSION THREAD

**Discussion by dwiyatci** [react-hook-form/react-hook-form#3681] [Q&A] [answered]
Hello, just a quick question to clarify. As per the `CHANGELOG` of [v6.11.0](https://github.com/react-hook-form/react-hook-form/blob/master/CHANGELOG.md#6110---2020-11-07):

> when `watch` with `useFieldArray`, `fields` object is no longer required as defaultValue

Does this rule also apply for its `useWatch` counterpart, i.e.

```diff
- useWatch({ name: 'fieldArray', control, defaultValue: fields });
+ useWatch({ name: 'fieldArray', control });
```

Thank you. :)

**Comment by bluebill1049** [comment, marked as answer]
hmmmm i didn't check `useWatch`, it should tho.
